docs/welcome/what-is-tartiflette.md

```diff
@@ -0,0 +1,29 @@
+---
+id: what-is-tartiflette
+title: What is Tartiflette
+sidebar_label: What is Tartiflette?
+---
+
+Welcome, glad to see you here.
+
+**Tartiflette** is a GraphQL Engine, built on top of Python 3.6 and higher. Focus on building GraphQL API based on the **Schema Definition Language** _(commonly called 'SDL')_.
```

Comment:
built on top of Python 3.6 and above. It's also focused on building GraphQL APIs using the awesome **Schema Definition Language**

Reply:
I removed "also" in your proposal.